only support for contiguous arrays

I have created my own build recipe using anaconda, I am using vtk 8.0 python bindings on latest tagged version 1.8.10

When I convert this mesh from gmsh format to vtu I get contiguous array error. 

```
meshio-convert  DSI020CALb.msh DSI020CALb.vtu
Number of points: 2116098
Elements:
  Number of triangles: 1410720
  Number of tetras: 11285760
Cell data: geometrical, physical
Traceback (most recent call last):
  File "/home/ksansom/anaconda3/envs/meshio/bin/meshio-convert", line 4, in <module>
    __import__('pkg_resources').run_script('meshio==1.8.10', 'meshio-convert')
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/pkg_resources/__init__.py", line 744, in run_script
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/setuptools-27.2.0-py3.6.egg/pkg_resources/__init__.py", line 1506, in run_script
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/EGG-INFO/scripts/meshio-convert", line 130, in <module>
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/EGG-INFO/scripts/meshio-convert", line 70, in _main
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/meshio/helpers.py", line 187, in write
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/meshio/vtk_io.py", line 236, in write
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/meshio/vtk_io.py", line 335, in _generate_vtk_mesh
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/vtk/util/numpy_support.py", line 132, in numpy_to_vtk
    assert z.flags.contiguous, 'Only contiguous arrays are supported.'
AssertionError: Only contiguous arrays are supported.
```
same kind of error when is use this command
```
./meshio-convert.py --input-format gmsh-binary --output-format vtu-binary DSI020CALb.msh DSI020CALb.vtu
Number of points: 2116098
Elements:
  Number of triangles: 1410720
  Number of tetras: 11285760
Cell data: geometrical, physical
Traceback (most recent call last):
  File "./meshio-convert.py", line 130, in <module>
    _main()
  File "./meshio-convert.py", line 70, in _main
    field_data=field_data
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/meshio/helpers.py", line 187, in write
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/meshio/vtk_io.py", line 236, in write
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/meshio-1.8.10-py3.6.egg/meshio/vtk_io.py", line 335, in _generate_vtk_mesh
  File "/home/ksansom/anaconda3/envs/meshio/lib/python3.6/site-packages/vtk/util/numpy_support.py", line 132, in numpy_to_vtk
    assert z.flags.contiguous, 'Only contiguous arrays are supported.
AssertionError: Only contiguous arrays are supported.
```

--- a/tools/meshio-convert
+++ b/tools/meshio-convert
@@ -4,7 +4,7 @@ Convert a mesh file to another.
 '''
 import meshio
 
-import numpy as np
+import numpy
 
 
 def _main():
@@ -38,11 +38,11 @@ def _main():
         # remove_orphaned_nodes.
         # find which nodes are not mentioned in the cells and remove them
         flat_cells = cells['tetra'].flatten()
-        orphaned_nodes = np.setdiff1d(np.arange(len(points)), flat_cells)
-        points = np.delete(points, orphaned_nodes, axis=0)
+        orphaned_nodes = numpy.setdiff1d(numpy.arange(len(points)), flat_cells)
+        points = numpy.delete(points, orphaned_nodes, axis=0)
         # also adapt the point data
         for key in point_data:
-            point_data[key] = np.delete(
+            point_data[key] = numpy.delete(
                     point_data[key],
                     orphaned_nodes,
                     axis=0
@@ -50,14 +50,17 @@ def _main():
 
         # reset GLOBAL_ID
         if 'GLOBAL_ID' in point_data:
-            point_data['GLOBAL_ID'] = np.arange(1, len(points)+1)
+            point_data['GLOBAL_ID'] = numpy.arange(1, len(points)+1)
 
         # We now need to adapt the cells too.
-        diff = np.zeros(len(flat_cells), dtype=flat_cells.dtype)
+        diff = numpy.zeros(len(flat_cells), dtype=flat_cells.dtype)
         for orphan in orphaned_nodes:
-            diff[np.argwhere(flat_cells > orphan)] += 1
+            diff[numpy.argwhere(flat_cells > orphan)] += 1
         flat_cells -= diff
         cells['tetra'] = flat_cells.reshape(cells['tetra'].shape)
+
+    # Some converters (like VTK) require `points` to be contiguous.
+    points = numpy.ascontiguousarray(points)
 
     # write it out
     meshio.write(

